.circleci/config.yml

```diff
@@ -405,6 +405,8 @@ jobs:
     executor:
       class: medium+
       name: sb_node_16_browsers
+    environment:
+        NODE_OPTIONS: --max_old_space_size=6144
```

Comment:
I don't know if this helped, but it failed once without and succeeded twice with it.  Might just be coincidence?